xdocs/theming-options.md

```
@@ -0,0 +1,131 @@
+11-color palette generator with API. Outputs in OKLCH format for Tailwind v4's `@theme` directive.
+
+**Tailwind Ink**  
+Neural network trained on Tailwind palette to generate color shades.
+
+---
+
+## CLI Tools
+
+**shadcn-custom-theme**  
+Generate custom themes from Tailwind color names:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)